docs/contributor/01-00-installation.md

```diff
@@ -0,0 +1,165 @@
+## Install in modular Kyma on the local k3d cluster
+
+1. Set up a local k3d cluster and a local Docker registry.
+
+```bash
+k3d cluster create kyma --registry-create registry.localhost:0.0.0.0:5001
+```
+
+2. Add the `etc/hosts` entry to register the local Docker registry under the name `registry.localhost`.
```

Comment:
Maybe we should use `k3d-kyma-registry` since it's also used by kyma cli when running `kyma provision k3d`